scripts/shutdown_all_serve.sh

```diff
@@ -0,0 +1 @@
+ps -eo user,pid,cmd|grep fastchat.serve|grep -v grep|awk '{print $2}'|xargs kill -9
```

Comment:
Could you merge all the sh files into one python script. and we can use it this way?
```
python shutdown.py --[all | controller | worker | api-server]
```

Reply:
With pleasure. It will come soon.